README.md

```diff
@@ -381,7 +390,7 @@ If you're opening issues related to these, please mention the version that the i
 License
 -------
 
-Proclaim is licensed under the [MIT][info-license] license.  
+Proclaim is licensed under the [MIT][info-license] license.
```

Comment:
Removal of this whitespace messes up README formatting. Could you add back or replace with an actual `<br/>`? I've been meaning to for ages 😅